resources/docker-compose.yaml

```diff
@@ -11,19 +11,25 @@ services:
       - adapter
       - enforcer
   adapter:
-    image: wso2/mg-adapter:4.0.0-m4
+    image: wso2/mg-adapter:4.0.0-m4-SNAPSHOT
     volumes:  
       - ./resources/adapter/security:/home/wso2/security
       - ./resources/conf:/home/wso2/conf
     ports:
       - "18000:18000" 
       - "9843:9843"
   enforcer:
-    image: wso2/mg-enforcer:4.0.0-m4
+    image: wso2/mg-enforcer:4.0.0-m4-SNAPSHOT
     volumes:  
       - ./resources/enforcer/security:/home/wso2/mg/security
       - ./resources/conf:/home/wso2/conf
+    links:
+      - adapter
     environment:
+      - ENFORCER_PRIVATE_KEY_PATH=/home/wso2/mg/security/mg.key
+      - ENFORCER_PUBLIC_KEY_PATH=/home/wso2/mg/security/mg.pem
+      - TRUSTED_ADAPTER_CERTS_PATH=/home/wso2/mg/security
```

Comment:
introduce separate folder for truststore and keystore

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/1582/commits/925259a8d10006fb5f6a5c57750be3a910e8410d